### PR TITLE
To-Do-List: Interactive feature

### PR DESCRIPTION
**Pull request by cosywasswa**
I performed the following tasks.
-Created a new branch called interactive-feature.
-Created a modules directory inside src directory
-Created interactive.js for the added functionalities.
-Created code for javascript in script.js and imported code from interactive.js.
-Styled elements accordingly
-Updated README file to include new features.
-Corrected linter errors.

### CODE REVIEW

File: src/modules/interactive.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    const index = Array.from(container.children).indexOf(listItem);
+
+    listItem.remove();
+    myTasks.splice(index, 1);
+  });
+
+  updateIndexes();
+  storeItems();
+};
+// eslint-disable-next-line no-unused-vars
```
Comment (mckent05) [inline src/modules/interactive.js]: - Please remove all disabled linter checks, remember linters help us ensure we write code that is clean and easy to read and maintain. Please fix this 👍 👍

File: src/modules/interactive.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const clearCompletedItems = (container, myTasks, updateIndexes, storeItems) => {
+  const completedItems = container.querySelectorAll('.completed');
+
+  completedItems.forEach((completedItem) => {
+    const listItem = completedItem.closest('.list');
+    const index = Array.from(container.children).indexOf(listItem);
+
+    listItem.remove();
+    myTasks.splice(index, 1);
```
Comment (mckent05) [inline src/modules/interactive.js]: - According to the project requirements, you are required to use the `Array.filter()` method to remove all completed items, please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @cosywasswa, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have done well ensuring that your project is interactive 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
Hi @cosywasswa , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a descriptive pull request summary
🚀 You implemented all the changes that the previous reviewer requested.
🚀 You used GitHub flow properly
🚀 You have no linter errors


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**